VideoWeb/VideoWeb.Common/Security/BaseServiceTokenHandler.cs

```diff
@@ -30,12 +30,12 @@ public abstract class BaseServiceTokenHandler : DelegatingHandler
         protected override async Task<HttpResponseMessage> SendAsync(HttpRequestMessage request,
             CancellationToken cancellationToken)
         {
-            var token = GetServiceToServiceToeken();
+            var token = GetServiceToServiceToken();
             request.Headers.Add("Authorization", $"Bearer {token}");
             return await base.SendAsync(request, cancellationToken);
         }
         
-        protected string GetServiceToServiceToeken()
+        protected string GetServiceToServiceToken()
```

Comment:
here the same!
----
this should not have been merged originally. ridiculous!

Reply:
okay done now